### DISCUSSION TITLE
Admin pages after v4.1.4/4.1.5

### DISCUSSION THREAD

**Discussion by stuartoram** [mastodon/mastodon#26124] [General]
After upgrading my instance to 4.1.4/4.1.5 I'm unable to load the admin pages in the web UI.
All else is functioning as expected from a user perspective.
Cannot see anything obvious in the logs I have checked.
Any pointers?

**Comment by renchap** [comment]
Can you look at your browser's inspector console and see if you have any errors here?

**Comment by stuartoram** [comment]
Appears to result in the 'oops' page although this doesn't load in the browser.

**Comment by renchap** [comment]
Is this similar to #26157 ?

**Comment by stuartoram** [comment]
Looks like it could well be, yes.
![image](https://github.com/mastodon/mastodon/assets/88890495/51d4eff9-22fc-4386-8090-b2365f0c2f8b)

**Reply by stuartoram** [reply]
Also just noticed this in logs, not sure if it's a factor but all except admin pages seems to be functioning normally.
![image](https://github.com/mastodon/mastodon/assets/88890495/3139faa3-a883-47a4-97e5-fa95eef7b34e)

**Reply by renchap** [reply]
This last error does not seem related.

We identified the problem in the issue linked above, you probably have a server configuration issue where your Puma host/container can not reach out to your media storage. Please see the comments on the issue for more details.

**Reply by stuartoram** [reply]
Sorry, thought I'd read all the comments on that earlier today, but clearly missed a few.
Not sure about the database error, but doesn't seem to be causing me any issues for now...so I'm going with don't fix what's not broken..
As far as the admin dashboard error, I've tried to play with hairpin NAT rules but not working out as expected, so I've added an entry to the hosts file for now and that's sorted it.
Thanks

Edit: Hairpin NAT fixed (typo), hosts file entry removed, still working fine.

**Reply by renchap** [reply]
Ok, Nice to know it is fixed for you then!

Closing the discussion as fixed.
